<!DOCTYPE html>
<html >
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <meta name="author" content="Byron J. Smith, PhD">
  <title>Byron J. Smith, PhD</title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-40659359-4', 'auto');
  ga('send', 'pageview');

</script>
  
</head>
<body style="max-width: 700px; margin-left: 80px; margin-right: 80px; font-family: arial,sans-serif,auto;">
<header>
<h1 class="title">Byron J. Smith, PhD</h1>
<div class="contact">
  /
  <a id="email-link" href="mailto:me@byronjsmith.com">me@byronjsmith.com</a> / 
  <a id="twitter-link" href="https://twitter.com/ByronJSmith">@ByronJSmith</a> / 
  <a id="linkedin-link" href="https://www.linkedin.com/in/bsmith89">LinkedIn</a> / 
  <a id="gscholar-link" href="https://scholar.google.com/citations?user=ZtIlAngAAAAJ">Google Scholar</a> / 
  <a id="orcid-link" href="https://orcid.org/0000-0002-0182-404X">ORCiD</a> / 
  <a id="github-link" href="https://github.com/bsmith89">Github</a> / 
  <a id="website-link" href="http://blog.byronjsmith.com">Website</a> / 
</div>
</header>
<hr>
<h2 id="research-focus">Research Focus</h2>
<p>I develop and use bioinformatic tools and statistical models to integrate diverse data towards understanding complex microbial communities.</p>
<h2 id="publications">Publications</h2>
<ul>
<li><p>[2022] C. Zhao, M. Goldman, <strong>B.J. Smith</strong>, K.S. Pollard. Genotyping microbial communities with MIDAS2: from metagenomic reads to allele frequency tables. <em>Current Protocols in Bioinformatics</em> doi:<a href="https://doi.org/10.1002/cpz1.604">10.1002/cpz1.604</a></p></li>
<li><p>[2022] <strong>B.J. Smith</strong>, X. Li, Z.J. Shi, A. Abate, K.S. Pollard. Scalable microbial strain inference in metagenomic data using StrainFacts. <em>Frontiers in Bioinformatics</em> doi:<a href="https://doi.org/10.3389/fbinf.2022.867386">10.3389/fbinf.2022.867386</a></p></li>
<li><p>[2022] <strong>B.J. Smith</strong>, Y.M. Piceno, M. Zydek, B. Zhang, L. Aboud Syriani, J.P. Terdiman, Z. Kassam, A. Ma, S.V. Lynch, K.S. Pollard, N. El-Nachef. Strain-resolved analysis in a randomized trial of antibiotic pretreatment and maintenance dose delivery mode with fecal microbiota transplant for ulcerative colitis. <em>Scientific Reports</em> doi:<a href="https://doi.org/10.1038/s41598-022-09307-5">10.1038/s41598-022-09307-5</a></p></li>
<li><p>[2021] <strong>B.J. Smith</strong>, R.A. Miller, T.M. Schmidt. <em>Muribaculaceae</em> genomes assembled from metagenomes suggest genetic drivers of differential response to acarbose treatment in mice. <em>mSphere</em> doi:<a href="https://doi.org/10.1128/msphere.00851-21">10.1128/msphere.00851-21</a></p></li>
<li><p>[2021] <strong>B.J. Smith</strong>, K.S. Pollard. Evaluation of Le et al.: Challenges and opportunities for using data to understand equitability in science. <em>Cell Systems</em> doi:<a href="https://doi.org/10.1016/j.cels.2021.08.005">10.1016/j.cels.2021.08.005</a></p></li>
<li><p>[2020] N. El-Nachef, <strong>B.J. Smith</strong>, Y. Piceno, M. Zydek, L. Syriani, J.P. Terdiman, A. Ma, Z. Kassam, K. Pollard, S. Lynch. Antibiotic pretreatment prior to fecal microbiota transplantation increases rates of remission in patients with mild to moderate ulcerative colitis: Results from a pilot randomized clinical trial. <em>Gastroenterology</em> (Digestive Disease Week Abstract) doi:<a href="https://doi.org/10.1016/S0016-5085(20)30716-2">10.1016/S0016-5085(20)30716-2</a></p></li>
<li><p>[2019] <strong>B.J. Smith</strong>, R.A. Miller, A.A. Ericsson, R. Strong, D.E. Harrison, T.M. Schmidt. Changes in the gut microbiome and fermentation products concurrent with enhanced longevity in acarbose-treated mice. <em>BMC Microbiology</em>. doi:<a href="https://doi.org/10.1186/s12866-019-1494-7">10.1186/s12866-019-1494-7</a></p></li>
<li><p>[2015] S.F. Stoddard, <strong>B.J. Smith</strong>, R. Hein, B.R.K. Roller, T.M. Schmidt. rrnDB: Improved tools for interpreting rRNA gene abundance in Bacteria and Archaea and a new foundation for future development. <em>Nucleic Acids Research</em>. doi:<a href="https://doi.org/10.1093/nar/gku1201">10.1093/nar/gku1201</a></p></li>
</ul>
<h2 id="posters-and-presentations">Posters and Presentations</h2>
<ul>
<li><p>[2023 (upcoming)] “Strain-resolved inference of microbial gene content in fecal microbiota transplantation to treat ulcerative colitis.” (<a href="https:byronjsmith.com/rainin2023_abstract.pdf">poster</a>) <strong>Kenneth Rainin Foundation Innovations Symposium</strong>, San Francisco, CA</p></li>
<li><p>[2023 (upcoming)] “Resolving strain-level gene content variation from large, metagenomic datasets” (<a href="https:byronjsmith.com/grc_micropop_2023_abstract.pdf">poster</a>) <strong>Microbial Population Biology Gordon Research Conference</strong>, Andover, NH</p></li>
<li><p>[2022] “Strain tracking and reconstruction in large metagenomic datasets.” (<a href="https://byronjsmith.com/qbc_retreat_2022_slides.pdf">invited talk</a>) <strong>Quantitative Biosciences Consortium Retreat</strong>, Santa Cruz, CA</p></li>
<li><p>[2022] “Strain-resolved bacterial genome reconstruction in large, metagenomic datasets” (<a href="https://byronjsmith.com/noyce_symposium_2022_poster.pdf">poster</a>) <strong>University of California Noyce Initiative Symposium</strong>, Santa Barbara, CA</p></li>
<li><p>[2022] “Strain-resolved inference of microbial gene content in large metagenomic datasets.” (<a href="https://byronjsmith.com/cshl_microbiome_2022_slides.pdf">talk</a> / <a href="https://byronjsmith.com/cshl_microbiome_2022_abstract.pdf">abstract</a>) <strong>CSHL - Microbiome Meeting</strong>, Cold Spring Harbor, NY</p></li>
<li><p>[2022] “Scaling microbial strain inference to tens of thousands of metagenomes with fuzzy genotypes.” (<a href="https://byronjsmith.com/imsi2022_slides_with_notes.pdf">invited talk</a>) Workshop on <strong>Multiscale Microbial Communities: Dynamical Models, Ecology, and One Health</strong>, Institute for Mathematical and Statistical Innovation, Chicago, IL</p></li>
<li><p>[2021] “Bacterial genotype deconvolution in shotgun metagenomic reads using fuzzy genotypes” (<a href="https://byronjsmith.com/probgen2021_poster.pdf">poster</a>) <strong>Probabilistic Modeling in Genomics</strong>, Cold Spring Harbor Lab</p></li>
<li><p>[2020] “Identifying and tracking bacterial strains in metagenomic libraries” (<a href="https://byronjsmith.com/toolstech2020.pdf">talk</a>) <strong>Tools &amp; Tech RIPS</strong>, The Gladstone Institutes, San Francisco, California</p></li>
<li><p>[2020] “Strain-level bacterial reconstruction and inference in patients receiving fecal microbiota transplantation for ulcerative colitis” (<a href="https://byronjsmith.com/ignite2020_slides.pdf">talk</a>) <strong>Microbiome Ignite Series</strong>, University of California, San Francisco</p></li>
<li><p>[2019] “Improved methods for strain-specific genome reconstruction” (<a href="https://byronjsmith.com/biohubconfab2019_slides.pdf">lightning talk</a>) <strong>Chan Zuckerberg Biohub Inter-lab Confab</strong>, San Francisco, California</p></li>
<li><p>[2019] “<em>Muribaculaceae</em> Genomes assembled from metagenomes suggest genetic drivers of differential response to acarbose treatment in mice” (<a href="https://byronjsmith.com/asm2019_poster.pdf">poster</a>), <strong>American Society for Microbiology - Microbe</strong>, San Francisco, California</p></li>
<li><p>“Changes in the gut microbiota and fermentation products associated with enhanced longevity in acarbose-treated mice.” Presented at:</p>
<ul>
<li>[2018] (invited talk) <strong>Pollard Lab Meeting</strong>, University of California, San Francisco, invited by Dr. Katherine S. Pollard</li>
<li>[2018] (<a href="https://byronjsmith.com/keystone18X3_poster.pdf">poster</a>) <strong>Keystone Symposium on Manipulation of the Gut Microbiota for Metabolic Health</strong>, Banff, Alberta</li>
<li>[2017] (invited talk) <strong>Lynch Lab Meeting</strong>, University of California, San Francisco, invited by Dr. Susan V. Lynch</li>
<li>[2017] (talk) <strong>Microbiome Group Seminar Series</strong>, University of Michigan</li>
<li>[2017] (poster) <strong>American Society for Microbiology - Microbe</strong>, New Orleans, Louisiana</li>
</ul></li>
<li><p>[2017] “Automating reproducible data analysis pipelines with Snakemake” (<a href="https://blog.byronjsmith.com/snakemake-analysis.html">invited talk</a>), <strong>Department of Research Computing - Joint User Groups Meeting</strong>, Boise State University</p></li>
<li><p>[2016] “The longevity enhancing drug Acarbose modulates the gut microbiome and its fermentation products” (poster), <strong>Michigan Meeting on the Microbiome</strong>, University of Michigan</p></li>
<li><p>[2014] “Slow but successful: spatial structure and the evolution of efficiency” (talk), <strong>Pre-Candidacy Seminar</strong>, Department of Ecology and Evolutionary Biology, University of Michigan</p></li>
<li><p>[2011] “GABA signaling and reception in the developing nervous system” (talk) <strong>Spring Honors Colloquium</strong>, The College of William &amp; Mary</p></li>
<li><p>[2010] “The role of calcium in neural phenotype differentiation in developing Xenopus” (poster), <strong>Undergraduate Research Symposium</strong>, The College of William &amp; Mary</p></li>
</ul>
<h2 id="grants-and-fellowships">Grants and Fellowships</h2>
<ul>
<li><p>[2021-2022] Computational Innovation Post-doctoral Fellowship for my proposal, “Strain-resolved microbiome-wide association studies for the discovery of hidden, microbial drivers of disease”, funded for $75,000</p></li>
<li><p>[2016-2018] Integrated Training in Microbial Systems (ITiMS) Fellowship co-advised by Dr. Thomas M. Schmidt and Dr. Aaron A. King</p></li>
<li><p>[2015-2016] Instrumental in preparing a UM Glenn Center for Aging Research grant which was funded for $50,000. Principal Investigator: Dr. Thomas M. Schmidt</p></li>
<li><p>[2013-2014] UM Ecology &amp; Evolutionary Biology Departmental Block Grant</p></li>
<li><p>[2011-2012] Excellence in BioMolecular Science Fellowship from the College of Natural Science, Michigan State University</p></li>
<li><p>[2008-2010] Howard Hughes Medical Institute Student Research Awards</p></li>
<li><p>[2008] Howard Hughes Medical Institute Freshman Research Grant</p></li>
</ul>
<h2 id="honors-and-awards">Honors and Awards</h2>
<ul>
<li><p>[2023] Distinguished Achievement in Scientific Leadership Award from the Gladstone Institute of Data Science and Biotechnology</p>
<ul>
<li>This award is granted to exceptionally collaborative scientists who take on leadership roles at Gladstone and demonstrate exemplary collaboration, mentorship, and integrity that reaches beyond their own lab</li>
</ul></li>
<li><p>[2011] Nominated for Michigan State University Distinguished Fellowship or University Enrichment Fellowship</p></li>
<li><p>[2007, 2008, 2009, 2011] The College of William &amp; Mary Dean’s List for Academic Excellence</p></li>
<li><p>[2008] Inducted into Alpha Lambda Delta and Phi Eta Sigma Academic Honor Societies</p></li>
</ul>
<h2 id="education-and-professional-experience">Education and Professional Experience</h2>
<ul>
<li>[2019-present] Postdoctoral Scholar
<ul>
<li><strong>Gladstone Institute</strong> for Data Science and Biotechnology, San Francisco, CA</li>
<li>Research Advisor: Dr. Katherine S. Pollard</li>
</ul></li>
<li>[2013-2018] PhD in Ecology and Evolutionary Biology
<ul>
<li><strong>University of Michigan</strong>, Ann Arbor, MI</li>
<li><a href="https://byronjsmith.com/dissertation.pdf">Dissertation</a>: “Integrated Analysis of the Gut Microbiota and Their Fermentation Products in Mice Treated with the Longevity Enhancing Drug Acarbose”</li>
<li>Embedded Masters in Ecology and Evolutionary Biology, granted in 2016</li>
<li>GPA: 4.0</li>
<li>Research Advisor: Dr. Thomas M. Schmidt</li>
</ul></li>
<li>[2011-2013] Graduate Student in Microbiology and Molecular Genetics
<ul>
<li><strong>Michigan State University</strong>, East Lansing, MI</li>
<li>GPA: 4.0</li>
<li>Affiliate, BEACON Center for The Study of Evolution in Action</li>
<li>Affiliate, Great Lakes Bioenergy Research Center</li>
<li>Research Advisor: Dr. Thomas M. Schmidt</li>
</ul></li>
<li>[2007-2011] Bachelor of Science in Biology
<ul>
<li><strong>The College of William &amp; Mary</strong>, Williamsburg, VA</li>
<li>Graduated <em>Summa Cum Laude</em></li>
<li>GPA: 3.80 (overall) / 4.00 (major)</li>
<li>Minor: Mathematics</li>
<li>Research Advisor: Dr. Margaret S. Saha</li>
</ul></li>
</ul>
<h2 id="research-training">Research Training</h2>
<ul>
<li><p>[2019-Present] Advised by <strong>Dr. Katherine S. Pollard</strong> at the University of California, San Francisco</p></li>
<li><p>[2012-2018] Advised by <strong>Dr. Thomas M. Schmidt</strong> at Michigan State University and University of Michigan</p></li>
<li><p>[2008-2011] Advised by <strong>Dr. Margaret S. Saha</strong> at The College of William &amp; Mary</p></li>
<li><p>[2006-2007] Advised by <strong>Dr. Mary A. Voytek</strong> at the United States Geological Survey</p></li>
</ul>
<h3 id="select-coursework-and-workshops">Select Coursework and Workshops</h3>
<ul>
<li><p>[2023] <strong>Scientific Leadership and Management</strong>, The Gladstone Institutes, San Francisco, CA</p>
<ul>
<li>Three-day course designed for senior postdocs and junior faculty to learn about “leadership styles, people management, inclusive leadership, and conflict resolution”.</li>
</ul></li>
<li><p>[2015] <strong>Train the Trainers: Teaching Adult Researchers to Program</strong>, Michigan State University</p></li>
<li><p>[2014] <strong>Guarda Workshop in Evolutionary Biology</strong>, University of Basel and ETH Zürich, Switzerland</p></li>
<li><p>[2012] <strong>Software Carpentry Bootcamp</strong>, Michigan State University</p></li>
<li><p>[2012] <strong>Enhancing Linkages Between Mathematics and Ecology</strong>, Kellogg Biological Station, Hickory Corners, MI</p></li>
<li><p>[2011] <strong>Microbial Metagenomics Workshop</strong>, Michigan State University</p></li>
</ul>
<h2 id="public-datasets-and-software">Public Datasets and Software</h2>
<ul>
<li><p><a href="https://github.com/bsmith89/StrainFacts">StrainFacts</a>: Computational tool to identify and track microbial strains across large numbers of metagenomic samples.</p></li>
<li><p><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA737472">PRJNA737472</a>: Longitudinal fecal metagenomes and 16S rRNA gene libraries from ulcerative colitis patients receiving fecal microbiota transplants with and without antibiotic pretreatment and with maintenance dosing via either capsules or enema.</p>
<ul>
<li>Associated bile acid profiles are also available</li>
</ul></li>
<li><p><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA448009">PRJNA448009</a>: Paired metagenomes and 16S rRNA gene libraries of fecal samples from acarbose treated and control mice, as well as carefully curated metagenome assembled genomes from nine members of the understudied bacterial family, the <em>Muribaculaceae</em></p>
<ul>
<li>Associated metabolite profiles and mouse longevity data are also available</li>
</ul></li>
</ul>
<h2 id="formal-mentorship">Formal Mentorship</h2>
<ul>
<li>[2023] <strong>Cindy Pino-Barrios</strong> (Rotation Student in Biological and Medical Informatics, UCSF)
<ul>
<li>Served as a co-primary mentor during her 10-week rotation in the Pollard Lab, where she initiated a project studying the use of single-nucleotide polymorphisms in the core genome to predict the presence/absence of functional gene families in a publicly available reference genome collection. This project has become the basis for a manuscript that is in preparation.</li>
<li>Currently: Rotation Student working with Prof. Catherine Tcheandjieu, The Gladstone Institutes</li>
</ul></li>
<li>[2023] <strong>Claire Dubin</strong> (Rotation Student in Bioinformatics, UCSF)
<ul>
<li>Served as a co-primary mentor during her 10-week rotation in the Pollard Lab, where she initiated a project tracking strain-resolved transmission outcomes in a large meta-study of fecal microbiota transplantation experiments.</li>
<li>Currently: Rotation Student at UCSF</li>
</ul></li>
<li>[2019] <strong>Erin Gilbertson</strong> (Rotation Student in Biological and Medical Informatics, UCSF)
<ul>
<li>Served as Erin’s primary mentor during her 10-week rotation in the Pollard Lab, where she worked on two separate studies during that time: (1) the analysis of a longitudinal, 16S rRNA amplicon data in a pilot study comparing anti-TNF and Vedolizumab treatments for ulcerative colitis patients, and (2) a simulation study benchmarking the performance of StrainFinder, software for the deconvolution of metagenotype data.</li>
<li>Currently: PhD Candidate working with Prof. Tony Capra, UCSF</li>
</ul></li>
</ul>
<h2 id="teaching">Teaching</h2>
<ul>
<li>[2015-present] Instructor, <strong>Software Carpentry Bootcamps</strong>,
<ul>
<li>Co-taught over a dozen workshops on computing skills for researchers: programming, the command-line version control, data management, etc.</li>
<li>Students are postdocs, graduate students, faculty, etc.</li>
<li>Workshops at Stanford University, Harvard Medical School, University of Pennsylvania, University of California, San Francisco, University of New Hampshire, University of Michigan, Wayne State University, Lawrence Berkeley National Laboratory, and Genentech</li>
<li>Contributed to development of shared teaching materials, notably lessons on the tools Git and Make.</li>
</ul></li>
<li>[2020] Guest Lecturer, <strong>BMI 206: Statistical Methods of Bioinformatics</strong>, University of California, San Francisco
<ul>
<li>Prepared and presented an interactive lesson titled <a href="https://byronjsmith.com/bmi206_bayesian.pdf"><em>Brief Introduction to Bayesian Inference</em></a> to first year PhD students in the Biological and Medical Informatics Graduate Program</li>
<li>Course Instructor: Dr. Katherine S. Pollard</li>
</ul></li>
<li>[2015, 2018] Graduate Student Instructor, <strong>Introductory Biology Lab: The Human Microbiome</strong>, University of Michigan
<ul>
<li>Assisted during three semesters in designing and running laboratory activities and lectures for a novel, research based, introductory biology lab</li>
<li>Course Instructors: Dr. Thomas M. Schmidt, Dr. Kristi MacCready, and Dr. Arvind Venkateraman</li>
</ul></li>
<li>[2017] Teaching Assistant, <strong>Data Intensive Biology Summer Research Institute</strong> hosted by the University of California, Davis
<ul>
<li>Assisted with week-long workshops on “Environmental Metagenomics” and “Non-Model Organism RNA-Seq”</li>
<li>Organizer: Dr. C. Titus Brown</li>
</ul></li>
<li>[2016] Instructor, <strong>Bodega Marine Laboratory Bioinformatics Workshop</strong>, hosted by the University of California, Davis
<ul>
<li>Invited to teach a half-day lesson titled <a href="https://blog.byronjsmith.com/make-analysis.html"><em>GNU Make for Reproducible Analysis Pipelines</em></a></li>
<li>Workshop Organizer: Dr. C. Titus Brown</li>
</ul></li>
<li>[2014] Teaching Assistant, <strong>Software Carpentry Bootcamp</strong>, University of Michigan
<ul>
<li>Assisted instructors teaching basic computing skills to early career scientists</li>
</ul></li>
<li>[2013] Graduate Student Instructor, <strong>General Ecology</strong>, University of Michigan
<ul>
<li>Course Instructor: Dr. Aaron A. King</li>
</ul></li>
<li>[2012] Teaching Assistant, <strong>Microbial Genomics Lab</strong>, Michigan State University
<ul>
<li>Course Instructor: Dr. Robert A. Britton</li>
</ul></li>
</ul>
<h2 id="service-and-outreach">Service and Outreach</h2>
<ul>
<li>[2019-present] <strong>Gladstone Postdoc Advisory Committee</strong> organizer and member
<ul>
<li>Advised Gladstone leadership and administration on postdoc issues; involved in planning and executing numerous activities and events</li>
<li>Lead postdoc advocacy efforts to increase representation, recruitment, and retention of postdocs from communities historically under-represented in the life sciences, which directly contributed to the creation and implementation of the Gladstone DEI Strategic Plan.</li>
</ul></li>
<li>[2012-present] <strong>Peer Reviewer</strong>
<ul>
<li>Refereed manuscripts submitted to <em>eLife</em>, <em>Microbiome</em>, <em>Cell Systems</em>, <em>PLoS Computational Biology</em>, <em>PLoS Biology</em>, <em>Scientific Reports</em>, <em>ISME</em>, and others.</li>
</ul></li>
</ul>
<ul>
<li><p>[2018-2023] <strong>Software Carpentry Curriculum Advisory Committee</strong></p>
<ul>
<li>[2022-2023] Committee Chair</li>
</ul></li>
<li><p>[2019] <strong>UCSF Library Reproducibility for Biomedical Researchers</strong> panelist</p></li>
<li><p>[2016-2018] <strong>University of Michigan Software Carpentry</strong> group member</p>
<ul>
<li>Actively involved in developing a community of Software and Data Carpentry instructors, learners, and advocates</li>
</ul></li>
<li><p>[2017] <strong>Models: At the Intersection of Data and Discovery.</strong></p>
<ul>
<li>Collaborated in proposing a meeting for the directors and students of the Burroughs Wellcome Fund’s Institutional Program Unifying Population and Laboratory Based Sciences</li>
<li>Held at the University of Michigan</li>
<li>$150,000 budget</li>
</ul></li>
<li><p>[2016-2017] <strong>Ecology and Evolutionary Biology Statistics Office Hours</strong> organizer</p>
<ul>
<li>Co-founded and organized monthly statistics help sessions for graduate students</li>
</ul></li>
<li><p>[2016] Graduate Student Representative to the Department of Ecology and Evolutionary Biology <strong>Executive Committee</strong> during the fall term</p></li>
<li><p>[2014-2015] Graduate Student Representative to the Department of Ecology and Evolutionary Biology <strong>Seminar Committee</strong></p></li>
<li><p>[2015] National Ocean Sciences Bowl, “<strong>Great Lakes Bowl</strong>” volunteer</p></li>
<li><p>[2013-2014] Michigan <strong>Xplore Engineering</strong> “Microbiology and Environmental Engineering” workshop team</p></li>
</ul>
<div class="permalink">
  <a href="http://www.byronjsmith.com/resume.html">
      Version: 2023-05-19
    </a>
</div>
</body>
</html>
